jbmc/regression/jbmc/classpath-two-jars/test_incorrect2.desc

```diff
@@ -0,0 +1,9 @@
+KNOWNBUG
+Test.class
+--classpath `../../../../scripts/format_classpath.sh libs/test.jar lib/test2.jar`
+^EXIT=0$
+^VERIFICATION SUCCESSFUL$
+--
+--
+This is incorrect as it would search for a Test.class file.
```

Comment:
I don't fully understand the comment. Is that the expected behaviour?

Reply:
Clarified and fixed.